app/src/main/java/co/develhope/meteoapp/features/tomorrow/TomorrowScreen.kt

```diff
@@ -27,6 +27,15 @@ class TomorrowScreen : Fragment() {
 
         setUpLinearView()
 
+        binding.arrowCardView.setOnClickListener {
+            if (binding.cardViewTomorrow.visibility == View.GONE) {
+                binding.cardViewTomorrow.visibility = View.VISIBLE
+                binding.arrowCardView.animate().rotation(180f).start()
+            } else {
+                binding.cardViewTomorrow.visibility = View.GONE
+                binding.arrowCardView.animate().rotation(0f).start()
+            }
+        }
```

Comment:
Maybe if we have code duplicated between TomorrowScreen and TodayScreen we can extract the code into a separate method and reuse it in both screens.